### PR TITLE
Create snapcraft.yaml for 7.0

### PR DESCRIPTION
**Pull request by rbhanda**

### CODE REVIEW

File: src/installer/pkg/snap/dotnet-runtime-7.0/snap/snapcraft.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+name: dotnet-runtime-70
+base: core18
+version: 7.0.0-preview.1.22076.8
+summary: Cross-Platform .NET Core Runtime.
```
Comment (jkotas) [inline src/installer/pkg/snap/dotnet-runtime-7.0/snap/snapcraft.yaml:4]: Our branding is .NET Runtime. We have dropped Core.

Also “cross-platform” feels redundant in this description

**Comment by jkotas** [comment]
https://github.com/dotnet/runtime/pull/49697 moved the snapcraft files to installer repo. Should this file be rather added there? If not, why do we want to add these back to dotnet/runtime repo?

**Comment by danmoseley** [comment]
@rbhanda are you still working on this?

**Comment by rbhanda** [comment]
I will close this for now and open once we have a clearer place to put the yaml files.

**Pull request closed without merging**